latest/.data

```diff
@@ -1,6 +1,6 @@
 Name                2_1
 Release             v2.1.2
 Seeds               bd57ba901668135b3498528c294a216d5ab63b31@167.71.69.162:26656
-Persistent peers    3f26a7d3269e352574e760d4b08d045aa40ebbf2@64.225.78.169:26656,e84b1b8cdb6563170466912f082829c2e4d28ab1@64.225.64.179:26656,01f615a56a7ba14d39e4eaf2bfd00a885cc4e629@64.227.113.198:26656,d6e07b5f0a69bd4d6f4e6e931f00fbcb992a8654@134.122.98.51:26656
```

Comment:
If you edit this you have to edit the entire latest folder as well, adding the `data.json`, `data.yml` files and also updating the rest of the data. If you do not edit this folder with the latest available data open an issue to update